_data/languages.yml

```diff
@@ -9,4 +9,3 @@ en:
 sv:
   work_tweet: Säkerhet är viktigt, @TWITTERHANDLE. Vänligen lägg till two factor authentication på er hemsida.
   progress_tweet: Tack för att ni arbetar på two factor authentication, @TWITTERHANDLE!
-  
```

Comment:
As per [.editorconfig](https://github.com/2factorauth/twofactorauth/blob/master/.editorconfig#L10) there should always be a new line at the end of the file. So line 12 should not be removed.